src/components/article/article.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import './article.css';
```

Comment:
```suggestion
import './Article.css';
```
css тоже с большой буквы